starter-code/index.html

```diff
@@ -2,21 +2,51 @@
 <html lang="en">
 <head>
   <title>Instagram</title>
+  <link rel="stylesheet" href="./css/main.css">
 </head>
 <body>
-  Instagram
+  
 
-  Capture and Share the World's Moments
+    <div class="veryTop">
+    <div class="position">
 
-  Instagram is a fast, beautiful and fun way to share your life with friends and family.
+        <div class= "phone">
+            <img  src="images/phones.png" alt="">
+          </div>
 
-  Take a picture or video, choose a filter to transform its look and feel, then post to Instagram &mdash; it's that easy. You can even share to Facebook, Twitter, Tumblr and more. It's a new way to see the world.
+      <header>
```

Comment:
Instead of the "header" tag you should use a regular "div", it will be easier to read your code later :)